src/pyload/plugins/containers/CCF.py

```diff
@@ -30,7 +30,7 @@ class CCF(BaseContainer):
     __description__ = """CCF container decrypter plugin"""
     __license__ = "GPLv3"
     __authors__ = [
-        ("Willnix", "Willnix@pyload.net"),
+        ("Willnix", "Willnix@pyload.org"),
```

Comment:
`pyload.org` is a scam site now, so use `pyload.net` always.

Reply:
Changed (forgot it, or did a wrong merge)